File: Book.java
public class Book {
    private String name;
    private String author;
    private boolean isRented = false;
    private String user;

    public Book(String name, String author) {
        this.name = name;
        this.author = author;
    }

    public String getName() {
        return name;
    }

    public String getAuthor() {
        return author;
    }

    public void setRented(boolean rented) {
        isRented = rented;
    }

    public void setUser(String user) {
        this.user = user;
    }

    public boolean isRented() {
        return isRented;
    }

    public String getUser() {
        return user;
    }
}

File: Book.py
class book:
    def __init__(self, name, author):
        self.name = name
        self.author = author
        self.isRented = False
        self.user = None

    def getName(self):
        return self.name

    def getAuthor(self):
        return self.author

    def setRented(self, rented):
        self.isRented = rented

    def setUser(self, user):
        self.user = user

    def isRented(self):
        return self.isRented

    def getUser(self):
        return self.user


